torch/_ops.py

```diff
@@ -198,12 +259,10 @@ def __str__(self):
         return "{}.{}.{}".format(*self._schema.name.split("::"), self._overloadname)
 
     def decompose(self, *args, **kwargs):
+        # TODO: should this consult py_impl?
         dk = torch._C.DispatchKey.CompositeImplicitAutograd  # type: ignore[attr-defined]
-        if (
-            torch._C._dispatch_has_kernel_for_dispatch_key(self.name(), dk)
-            or dk in self.py_kernels
-        ):
```

Comment:
what happened to py_kernels?

Reply:
I guess I can restore consulting it

Reply:
which kernels is self.py_kernels anyway, is that where non-dispatcher metas go or something?  i wasn't sure why you removed it but also not sure what it's for.